opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml

```diff
@@ -0,0 +1,51 @@
+#Minimal configuration of a cluster with version 2.X of the operator.
+#Note the replacement of 'master' role with 'cluster_manager' on line 49
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+metadata:
+  name: my-first-cluster
+  namespace: default
+spec:
+  security:
+    config:
+    tls:
+       http:
+         generate: true 
+       transport:
+         generate: true
+         perNode: true
+  general:
+    httpPort: 9400
+    serviceName: my-first-cluster
+    version: 2.3.0
+    pluginsList: ["repository-s3"]
+    drainDataNodes: true
+  dashboards:
+    tls:
+      enable: true
+      generate: true
+    version: 2.3.0
+    enable: true
+    replicas: 1
+    resources:
+      requests:
+         memory: "512Mi"
+         cpu: "200m"
+      limits:
+         memory: "512Mi"
+         cpu: "200m"
+  nodePools:
+    - component: masters
+      replicas: 3
+      resources:
+         requests:
+            memory: "4Gi"
```

Comment:
Resources are quite high for a minimal configuration (the other example has 2Gi and 500m)

Reply:
I got a pod eviction due to 137 memory error on IKS with the other example, numerous times.

Reply:
I tried numerous times with the lower resources on IKS but the pod was repeated killed by the scheduler for exceeding memory limits (137 status).